helpers/cf_helper.py

```diff
@@ -65,7 +61,7 @@ def stack_was_created_successfully(self, stack_name, attempt=1):
         except Exception as e:
             logging.info('There was a problem checking status of stack: {0}'.format(e))
         logging.info('Stack creation still in progress')
-        time.sleep(20)
+        time.sleep(sleep_time)
```

Comment:
I don't believe it is necessary to use an argument here for sleep_time. The application will always check status of stack creation every 20 seconds. 
It can just be a variable inside the method
```python
sleep_time = 20
```
If you want to be more explicit, you can add sleep_time to logging info in line above...
```python
 logging.info('Stack creation still in progress. Waiting {0} seconds'.format(sleep_time))
```